### DISCUSSION TITLE
heatmap_pal

### DISCUSSION THREAD

**Discussion by altoidaddict** [PlummersSoftwareLLC/NightDriverStrip#74] [Q&A]
anyone else had a hard time finding the correct reference to heatmap_pal?  heatmap_pal is plugged in by default to several of the effects but can't seem to find it defined anywhere

**Comment by MrCmonster** [comment]
The same goes for `purpleflame_pal` It isn't defined anywhere but is referenced in `FIRESTICK` and `UMBRELLA` in effects.cpp
